### ISSUE TITLE
Invalid method

### ISSUE THREAD

**Issue by Ttech** [nccgroup/featherduster#80]
There appears to be a broken method in hash extend that is not valid due to it missing a def ():
In addition to this, there appears to be no body for the method.

https://github.com/nccgroup/featherduster/blob/52ed10dc441c45649ce4f4bedbf59bb72c63cda8/feathermodules/hash/hash_extend.py#L11

**Comment by unicornsasfuel** [comment]
Yeah, this is dead code, but it's also not included in `__all__` in `featherduster/feathermodules/hash/__init__.py`, so it shouldn't affect the function of FeatherDuster currently.

It's a valid issue, but a very low severity one.
